src/Google/AdsApi/Common/AdsHeaderFormatter.php

```diff
@@ -118,11 +118,20 @@ private function formatUtilUsages($adsUtilities, $includeUtilityUsage)
 
     private function formatGuzzleInfo()
     {
-        $guzzleInfoTokens = ['GuzzleHttp/' . ClientInterface::VERSION];
+        $guzzleInfoTokens = ['GuzzleHttp/' . $this->getGuzzleVersion()];
         if (extension_loaded('curl') && function_exists('curl_version')) {
             $guzzleInfoTokens[] = 'curl/' . \curl_version()['version'];
         }
 
         return implode(', ', $guzzleInfoTokens);
     }
+
+    private function getGuzzleVersion()
+    {
+        if (defined('GuzzleHttp\ClientInterface::MAJOR_VERSION')) {
+            return ClientInterface::MAJOR_VERSION;
+        }
+
+        return ClientInterface::VERSION;
```

Comment:
This is not compilable as the new version of Guzzle doesn't have this constant any more.

Please remove the function and simply replace line 121 with:
        $guzzleInfoTokens = ['GuzzleHttp/' . ClientInterface::MAJOR_VERSION];

Reply:
This change was made in order to support both v6 and v7 of Guzzle, meaning that those you won't or can't update Guzzle are still able to use the package.

If keeping v6 support isn't desired, then I can of course make the change you've requested

Reply:
I see your intent (thanks), but we should support only one version.